src/modules/0.0.19/aws_cloudfront/entity/distribution.ts

```diff
@@ -70,11 +70,18 @@ export class Distribution {
     DomainName: string | undefined;
     Id: string | undefined;
     OriginShield: any;
-    CustomOriginConfig: {
-      HTTPPort: number | undefined;
-      HTTPSPort: number | undefined;
-      OriginProtocolPolicy: originProtocolPolicyEnum;
-    };
+    CustomOriginConfig:
+      | {
```

Comment:
What does a prefixed `|` in a type definition do?

Reply:
it's a way of adding multiline i believe, it is just added by prettier